cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -177,17 +177,19 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
       } );
 
       var inputData = {
-        'id':        option.value,
+        'id':     option.value,
         // Type must come before value or IE fails
-        'type':    'checkbox',
-        'value':   option.value,
-        'name':    _name,
-        'class':   'cf-input cf-multi-select_checkbox',
-        'inside':  li
-      }
+        'type':   'checkbox',
+        'value':  option.value,
+        'name':   _name,
+        'class':  'cf-input cf-multi-select_checkbox',
+        'inside': li
```

Comment:
Yeah, I quoted some of them because there were some with hyphens, meant to go back and update them all but lost track of it. Not sure which is best, but I'd say they should at least be consistent within each object.